boards/nxp/frdm_mcxn947/frdm_mcxn947-pinctrl.dtsi

```diff
@@ -31,6 +31,16 @@
 		};
 	};
 
+	pinmux_flexcomm2_lpuart: pinmux_flexcomm2_lpuart {
+		group0 {
+			pinmux = <FC2_P0_PIO4_0>,
+				<FC2_P1_PIO4_1>;
+			slew-rate = "fast";
+			drive-strength = "low";
+			input-enable;
+		};
+	};
```

Comment:
Should this be in the test overlay.

Reply:
Same comment as above- with the update to support I2C and UART on the same LPFLEXCOMM, this should be ok to be in the board file